src/game_api/math.hpp

```diff
@@ -283,6 +283,26 @@ struct Quad
         return *this;
     }
 
+    Quad& flip_horizontal()
```

Comment:
Shouldn't this be `flip_horizontally` to match with the other function?
or is this some English exception that i don't know of?